src/vulkan-renderer/renderer.cpp

```diff
@@ -105,18 +114,23 @@ void VulkanRenderer::render_frame() {
     m_frame_graph->render(image_index, m_rendering_finished_semaphore->get(), m_image_available_semaphore->get(),
                           m_vkdevice->graphics_queue());
 
+    m_imgui_overlay->render(image_index);
+
     // TODO(): Create a queue wrapper class
     auto present_info = wrapper::make_info<VkPresentInfoKHR>();
+
     present_info.swapchainCount = 1;
     present_info.waitSemaphoreCount = 1;
     present_info.pImageIndices = &image_index;
     present_info.pSwapchains = m_swapchain->swapchain_ptr();
     present_info.pWaitSemaphores = m_rendering_finished_semaphore->ptr();
+
```

Comment:
I personally don't really like the extra newlines here (subject to debate though I guess).

Reply:
I removed the top one. We should discuss this in the scope of `clang-format`.

Reply:
Done.